_posts/2015-09-05-neo-hpsrt-jekyll-template.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Neo-HPSTR Jekyll Blog template
```

Comment:
One more thing, is the theme name "Neo-HPSTR Jekyll Blog template" or "Neo-HPSTR Jekyll template"? If the latter, then change the title please and rename thumbnail image to neo-hpsrt-jekyll-template.png, i.e keep the thumbnail filename same as the template post, but without the date prefix.